lib/cretonne/src/ir/dfg.rs

```diff
@@ -106,6 +106,23 @@ impl DataFlowGraph {
     }
 }
 
+/// Resolve value aliases.
+///
+/// Find the original SSA value that `value` aliases.
+fn resolve_aliases(values: &EntityMap<Value, ValueData>, value: Value) -> Value {
+    let mut v = value;
+
+    // Note that extended_values may be empty here.
```

Comment:
Oops, `extended_values` is the old name for the `values` table back from when there was two kinds of values.